packages/core/src/components/block/block.tsx

```diff
@@ -26,7 +44,32 @@ export class TdsBlock {
     return level;
   }
 
+  private ensureKeyboardAccessibility() {
+    const childElements = Array.from(this.host.children);
+
+    childElements.forEach((child) => {
+      const isInteractive =
+        child instanceof HTMLButtonElement ||
+        child instanceof HTMLAnchorElement ||
+        child instanceof HTMLInputElement ||
+        child instanceof HTMLSelectElement ||
+        child instanceof HTMLTextAreaElement ||
+        child.getAttribute('tabindex') !== null ||
+        child.getAttribute('role') === 'button' ||
+        child.getAttribute('role') === 'link';
+
+      if (!isInteractive) {
+        child.setAttribute('tabindex', '-1');
+      }
+    });
+  }
```

Comment:
Let's imagine a situation where user nests tds-button or tds-text-field in tds-block, does it mean that script will set the tabindex to -1 and make them inaccessible? 
Please test this case.